src/server.ts

```diff
@@ -0,0 +1,52 @@
+import express, { Express } from 'express';
+import morgan from 'morgan';
+import helmet from 'helmet';
+import cors from 'cors';
+import config from '../config.json';
+import { getFilesWithKeyword } from './utils/getFilesWithKeyword';
+
+const app: Express = express();
+
+/************************************************************************************
+ *                              Basic Express Middlewares
+ ***********************************************************************************/
+
+app.set('json spaces', 2);
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
+// Handle logs in console during development
+if (process.env.NODE_ENV === 'development' || config.NODE_ENV === 'development') {
+  app.use(morgan('dev'));
+  app.use(cors());
+}
+
+// Handle security and origin in production
+if (process.env.NODE_ENV === 'production' || config.NODE_ENV === 'production') {
```

Comment:
@ffleandro what about disabling on development instead enabling on production?

Reply:
Not sure how to disable:

```
// Handle logs in console during development
if (process.env.NODE_ENV === 'development' || config.NODE_ENV === 'development') {
  app.use(morgan('dev'));
  app.use(cors());
}

// Handle security and origin in production
if (process.env.NODE_ENV === 'production' || config.NODE_ENV === 'production') {
  app.use(helmet());
}
```

What is your proposal?

Reply:
I've missed the part above however my proposal would be something like this:
```
switch(`${process.env.NODE_ENV || "production"}:${config.NODE_ENV || "production"}`) { 
   case 'development:development': { 
      app.use(morgan('dev'));
      app.use(cors());
      break; 
   } 
   case 'production:production': { 
      app.use(helmet());
      break; 
   }
   default: { 
      // throw some error
   } 
} 
```

Reply:
I've missed the part above however are inconsistent configuration values possible?
One set to production and one to development?